### DISCUSSION TITLE
Bot is @ tagging someone due to the package having his username in it

### DISCUSSION THREAD

**Discussion by The-Code-Monkey** [renovatebot/renovate#17951] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

So my packages are under an org @ techstack and the bot is accidentally tagging the user @techstack every time it creates a PR to one of my repos meaning he's getting a little annoyed is there a way to maybe reduce the chance of this happening.

https://github.com/The-Code-Monkey/PaperCMS

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment]
Can you link to a specific PR?

**Comment by The-Code-Monkey** [comment]
https://github.com/The-Code-Monkey/PaperCMS/pull/51 @rarkins

**Comment by rarkins** [comment, marked as answer]
The only link I see which is to `github.com/techstack` is from the commit:

<img width="333" alt="image" src="https://user-images.githubusercontent.com/6311784/192020025-13168e6a-1e62-4f17-8c30-4695a13eedb8.png">

Renovate cannot control whether GitHub generates that link.

It can only control the commit message, and it's you that's told Renovate to use the `@techstack` string in the groupName: https://github.com/The-Code-Monkey/PaperCMS/blob/7f539326b7e1d8d77b69035e63a6297aa6b75aa7/renovate.json#L20

If you removed the `@` symbol from that line, I think your problem might go away. If so, this is not a bug and is due to configuration.

**Reply by TechStack** [reply]
@The-Code-Monkey  This is how we resolved the mention when it happened previously.

**Reply by The-Code-Monkey** [reply]
Unable to do this as I believe it's the dependency dashboard where the issue is happening

**Reply by rarkins** [reply]
You need to identify where exactly the markdown is

**Comment by The-Code-Monkey** [comment]
Ok cheers Ill keep you updated once I fix that

**Reply by rarkins** [reply]
I have converted this issue to a discussion instead, as it does not appear to be a bug

**Comment by TechStack** [comment]
This appears to be happening again.

**Comment by The-Code-Monkey** [comment]
@rarkins I believe it's on this issue where it happens when the bot is triggering the mentions https://github.com/The-Code-Monkey/styled-system/issues/4

**Reply by rarkins** [reply]
And where exactly in the markdown? This was user error the last time

**Comment by The-Code-Monkey** [comment]
I believe it's when pr creation is rate limited there is just text which I
think is what causes the problem

**Reply by rarkins** [reply]
Like I said last year, you need to change this line: https://github.com/The-Code-Monkey/styled-system/blob/df827f479a1397709243cb5d562a0b038f8de0ac/renovate.json#L23

Just drop the `@`
